packages/vaex-ml/vaex/ml/sklearn.py

```diff
@@ -230,7 +230,7 @@ def fit(self, df, target, progress=None, **kwargs):
                     y = y[shuffle_index]
 
                 # train the model
-                self.model.partial_fit(X, y)
+                self.model.partial_fit(X, y, **kwargs)
```

Comment:
Note that if we want to support refit, we cannot add extra kwargs to fit, all fit methods should have the same arguments. I think it is weird anyway to have extra arguments, for sklearn as well, why not have that as extra propery/traits?

Reply:
I vote for having a 'partial_fit_kwargs' trait.
I think we should work towards a fixed fit method signature:
`def fit(self, df, progress=False):`
Any other arguments should be put as traits, so they become part of the state, and we can implement a refit, do online learning in the future etc.